vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -465,7 +465,7 @@ private void unstashWorkspace(config, prefix) {
         throw e
     } finally {
         echo "invalidate stash ${prefix}-${config.uniqueId}"
-        stash name: "${prefix}-${config.uniqueId}", excludes: '**/*', allowEmpty: true
+        new Utils().stash name: "${prefix}-${config.uniqueId}", excludes: '**/*', allowEmpty: true
```

Comment:
Also here we should think about re-using the already existing utils instance (... which needs to be forwarded into that method)

Reply:
https://github.com/SAP/jenkins-library/pull/4380#discussion_r1206678701

Reply:
Done: https://github.com/SAP/jenkins-library/pull/4380/commits/41837b0eaec45b6519995ab5afe0f0ad64ab29e0